spryker/usr/local/share/env/30-framework

```diff
@@ -31,9 +31,15 @@ export YVES_HOST_PROTOCOL=${YVES_HOST_PROTOCOL:-https://}
 export ZED_HOST=${ZED_HOST:-zed}
 export ZED_HOST_PROTOCOL=${ZED_HOST_PROTOCOL:-https://}
 
+export YVES_WEB_SERVER_NAME=${YVES_WEB_SERVER_NAME:-"yves $YVES_HOST"}
+export ZED_WEB_SERVER_NAME=${ZED_WEB_SERVER_NAME:-"zed $ZED_HOST"}
```

Comment:
I'm not sure what you mean. If YVES_HOST points at 127.0.0.1 and is used in the server name, it would be the same as `yves` pointing to 127.0.0.1 and used in the server name